Formula/tfschema.rb

```diff
@@ -0,0 +1,23 @@
+class Tfschema < Formula
+  desc "Terraform Pull Request Automation tool"
+  homepage "https://www.runatlantis.io/"
```

Comment:
These don't seem to match what this formula does, perhaps you used `atlantis` as a template and forgot to change this?

Reply:
yeah, duplicate issue. 😮‍💨